src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace Microsoft.Identity.Client
 {
-    internal sealed class ApplicationConfiguration : IAppConfigInternal
+    internal sealed class ApplicationConfiguration : IAppConfig
     {
         public const string DefaultClientName = "UnknownClient";
         public const string DefaultClientVersion = "0.0.0.0";
```

Comment:
IAppConfig is a public interface (in hindsight, a bad decision, becasuse it can't evolve), so keeping it.